flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/FlinkOperatorConfiguration.java

```diff
@@ -129,6 +131,14 @@ public static FlinkOperatorConfiguration fromConfiguration(Configuration operato
         String labelSelector =
                 operatorConfig.getString(KubernetesOperatorConfigOptions.OPERATOR_LABEL_SELECTOR);
 
+        SavepointFormatType savepointFormatType =
+                SavepointFormatType.valueOf(
+                        operatorConfig
+                                .getString(
+                                        KubernetesOperatorConfigOptions
+                                                .OPERATOR_SAVEPOINT_FORMAT_TYPE)
+                                .toUpperCase());
+
```

Comment:
This setting should not be part of the `FlinkOperatorConfiguration` is it is intended to be set on a per resource basis